db/migrations/00001_create_nodes_table.sql

```diff
@@ -6,7 +6,7 @@ CREATE TABLE eth_nodes
     genesis_block VARCHAR(66),
     network_id    NUMERIC,
     eth_node_id   VARCHAR(128),
-    UNIQUE (genesis_block, network_id, eth_node_id)
+    UNIQUE (genesis_block, network_id, eth_node_id, client_name)
 );
```

Comment:
Perhaps worth modifying the go.mod to point at the branch with these changes (unless they're already on staging?) so that CI verifies any other changes on the vdb side avoid causing any test failures